NitroxClient/GameLogic/Items.cs

```diff
@@ -111,7 +116,7 @@ public static IEnumerable<Entity> GetPrefabChildren(GameObject gameObject, Nitro
 
                     if (metadata.HasValue)
                     {
-                        NitroxId id = NitroxEntity.GetId(prefab.gameObject);
+                        NitroxId id = NitroxEntity.GetIdOrGenerateNew(prefab.gameObject);
```

Comment:
Should we do this before calling meta data extractors above?  Someone may introduce a metadata extractor that needs the id for something.